async-inference/HuggingFace-Async-Inference-Walkthrough.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        region="eu-west-1",
</pre><p>Use the <code>region</code>  variable set earlier as <code>region = sagemaker.Session().boto_region_name</code>	, rather than hard-coding region.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2910/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
fixed